### ISSUE TITLE
Question: Is this *USA* weather?

### ISSUE THREAD

**Issue by sroller** [jeremycole/weather_gov#21]

**Comment by jeremycole** [comment]
Yes, this is an API to weather.gov which only provides data for the USA (for the most part). The Ruby library works fine with any lat/lon but the weather.gov service will reject any lat/lon outside the US boundaries (mostly).

**Comment by sroller** [comment]
thanks.
